### DISCUSSION TITLE
COMO BLOQUEAR CORREOS EN MI CORREO DE CONFIRMACION

### DISCUSSION THREAD

**Discussion by RaulGuereca** [LaunchX-InnovaccionVirtual/CursoIntroPython#313] [Q&A]
Me estan llegando muchos correos de respuestas de github a mi correo que puse  de cofirmacion que tengo que hacer para que no me suceda esto ?

**Comment by JavierMendez-Coder** [comment]
Buenas noches. 

En el pie de página de tu correo debería aparecer algo similar a esto:

![image](https://user-images.githubusercontent.com/99059210/154793410-1162b139-3751-4cf7-b409-2d5cf52348bd.png)

Das click en desuscribir o unsubscribe, o puedes ingresar a la discusión y a la izquierda debería aparecer este botón:

![image](https://user-images.githubusercontent.com/99059210/154794131-30c28064-da4c-4a6e-b344-e1923c9bb02d.png)

Das click y listo.
